keyboards/crkbd/keymaps/vlukash_trackpad_left/rules.mk

```diff
@@ -0,0 +1,33 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
You only need to specify the options that differ from the base rules.mk here, so most of these can go.

Reply:
Fixed both rules.mk, thanks!